### PR TITLE
Butchering and masks

### PR DESCRIPTION
**Pull request by UndeadNappist**
-You can now butcher cazadors, ghouls, and deathclaws for leather
-Decreased meat drop on deathclaws
-Adds more leather types
-Adds mask spawn to raiders on a 10% chance.

### CODE REVIEW

**Comment by UndeadNappist** [comment]
Un-comments out the very silly punished raider role, and cleans up a few mask items.

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -276,8 +288,8 @@ Preacher
 /*
 Punished Raider
 */
-/*
-/datum/job/f13Punraider
+
+/datum/job/f13punraider
```
Comment (AndrewL97) [inline code/modules/jobs/job_types/wasteland.dm]: set positions to 0 if we are going to uncomment this - make it so only admins can activate it

File: code/modules/clothing/masks/miscellaneous.dm [hunk 2]
```diff
@@ -94,6 +103,7 @@
 	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
 	w_class = WEIGHT_CLASS_SMALL
 	var/voicechange = 1
+	item_flags = NODROP
```
[the remark this answers is not in the document]
Comment (UndeadNappist) [reply code/modules/clothing/masks/miscellaneous.dm]: Technically a bugfix for /spig/

[the remark this answers is not in the document]
Comment (AndrewL97) [reply code/modules/clothing/masks/miscellaneous.dm]: no drop is funny

**Pull request merged**